### PR TITLE
remove OEP and fix SEED tech allies link

### PR DESCRIPTION
**Pull request by kflemin**

### CODE REVIEW

File: src/app/tool/seed/faq/faq.component.html [hunk 1]
```diff
@@ -117,7 +116,7 @@ <h4 class="mat-h4 mb-4">What is the estimated cost to implement and maintain the
         <li><a href="https://opentech.eco/" target="blank">Open</a></li>
         <li><a href="https://psdconsulting.com/" target="blank">PSD Consulting</a>.</li>
```
Comment (nllong) [inline src/app/tool/seed/faq/faq.component.html:117]: should we remove PSD here as well since OEP was their connection to SEED?

**Review by nllong — CHANGES_REQUESTED** [review]
one small question...

File: src/app/tool/seed/faq/faq.component.html (hunk 1 above)
Comment (kflemin) [reply src/app/tool/seed/faq/faq.component.html:117]: sure!

Comment (kflemin) [inline src/app/tool/seed/faq/faq.component.html:117]:
```suggestion
```

**Review by nllong — APPROVED** [review]
perfetto.

**Pull request merged**